Shared/Samples/Find nearest vertex/FindNearestVertexView.swift

```diff
@@ -0,0 +1,198 @@
+            return Graphic(symbol: symbol)
+        }()
+        
+        /// A red diamond graphic for the nearest coordinate point.
+        let nearestCoordinateGraphic: Graphic = {
+            let symbol = SimpleMarkerSymbol(style: .diamond, color: .red, size: 10)
+            return Graphic(symbol: symbol)
+        }()
+        
+        /// The example polygon near San Bernardino County, California.
+        private var polygon: Polygon
+        
+        /// A distance formatter to format distance measurements and units.
+        private let distanceFormatter: MeasurementFormatter = {
+            let formatter = MeasurementFormatter()
+            formatter.numberFormatter.maximumFractionDigits = 1
+            formatter.numberFormatter.minimumFractionDigits = 1
+            return formatter
+        }()
+        
+        /// A location callout placement.
+        @Published var calloutPlacement: CalloutPlacement?
+        
+        /// The tap location.
+        @Published var tapLocation: Point!
+        
+        /// The nearest coordinate on the polygon to the tapLocation.
+        @Published var nearestCoordinateDistance: String = ""
+        
+        /// The nearest vertex on the polygon to the tapLocation.
+        @Published var nearestVertexDistance: String = ""
+        
+        init() {
+            // The spatial reference for the sample.
+            let statePlaneCaliforniaZone5 = SpatialReference(wkid: WKID(2229)!)!
+            
+            // Create the example polygon.
+            polygon = {
+                let polygonBuilder = PolygonBuilder(spatialReference: statePlaneCaliforniaZone5)
+                polygonBuilder.add(Point(x: 6627416.41469281, y: 1804532.53233782))
+                polygonBuilder.add(Point(x: 6669147.89779046, y: 2479145.16609522))
+                polygonBuilder.add(Point(x: 7265673.02678292, y: 2484254.50442408))
+                polygonBuilder.add(Point(x: 7676192.55880379, y: 2001458.66365744))
+                polygonBuilder.add(Point(x: 7175695.94143837, y: 1840722.34474458))
+                return polygonBuilder.toGeometry()
+            }()
+            
+            // Create the map.
+            map = {
+                let map = Map(spatialReference: statePlaneCaliforniaZone5)
+                
+                // Center map on the example polygon.
+                map.initialViewpoint = Viewpoint(
+                    center: polygon.extent.center,
+                    scale: 8e6)
+                
+                // Add US states feature layer to the map.
+                let usStatesGeneralizedLayer = FeatureLayer(
+                    item: PortalItem(
+                        portal: .arcGISOnline(connection: .anonymous),
+                        id: .usStatesGeneralized))
+                map.addOperationalLayer(usStatesGeneralizedLayer)
+                
+                return map
+            }()
```

Comment:
Please flatten the construction of `map`, or move it outside the `init()`. IMO [Using closure to set a property value](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/initialization/#Setting-a-Default-Property-Value-with-a-Closure-or-Function) is just a shortcut to avoid the real initializer. Same for the polygon.